package.json

```diff
@@ -74,7 +74,7 @@
     "@web3-react/fortmatic-connector": "^6.1.6",
     "@web3-react/injected-connector": "^6.0.7",
     "@web3-react/portis-connector": "^6.1.9",
-    "@web3-react/walletconnect-connector": "^6.2.4",
+    "@web3-react/walletconnect-connector": "^6.2.8",
```

Comment:
Is this related to the fix? happy to change versions, but not sure if is related

Reply:
Yes as far as I know the reason for the random QR code was the previous version. Since updating the version that issue didn't occur for me.